src/main.rs

```diff
@@ -292,12 +297,14 @@ upload-addr = \"{}/{}\"
         // assert!(components.iter().any(|s| s.starts_with("rustfmt-")));
         // assert!(components.iter().any(|s| s.starts_with("rls-")));
         // assert!(components.iter().any(|s| s.starts_with("clippy-")));
+
+        Ok(())
     }
 
-    fn download_artifacts(&mut self, rev: &str) {
+    fn download_artifacts(&mut self, rev: &str) -> Result<(), Error> {
```

Comment:
I personally prefer `anyhow::Result<()>` and not importing Error directly, but doesn't really matter.

Reply:
Changed the code to import `Result`.

Reply:
Does that look better to you?

Reply:
I specifically usually leave the anyhow:: prefix on it, since I find a bare Result confusing... I would probably rank them from best to worst (in my opinion, anyway):

 * `anyhow::Result<T>`
 * `Result<T, Error>` with `use anyhow::Error;`
 * `Result<T>` with `use anyhow::Result;`

Reply:
Ok, then I think I prefer to keep `Result<T, Error>` with `use anyhow::Error`.